.gitmodules

```diff
@@ -13,3 +13,7 @@
 [submodule "micro_ros_can/micro_ros_renesas2estudio_component"]
 	path = micro_ros_can/micro_ros_renesas2estudio_component
 	url = https://github.com/micro-ROS/micro_ros_renesas2estudio_component
+[submodule "micro_ros_tcp_freertos/micro_ros_renesas2estudio_component"]
+	path = micro_ros_tcp_freertos/micro_ros_renesas2estudio_component
+	url = https://github.com/micro-ROS/micro_ros_renesas2estudio_component
+	branch = feature/pmod_wifi
```

Comment:
TODO: modify submodule branch one https://github.com/micro-ROS/micro_ros_renesas2estudio_component/pull/30 is merged